product_docs/docs/pgd/4/deployments/tpaexec/installing_tpaexec.mdx

```diff
@@ -28,17 +28,31 @@ You need your token to complete the install process. It is available from the [E
 
 The examples in this section assume you are using TPAexec v22.14 and installing Postgres 14.
 
-1. Install the EnterpriseDB repository for TPAexec. Replace `token` with your company token. Replace `pgVersion` with the version of Postgres you are installing.
+1. Install the EnterpriseDB repository for TPAexec. 
 
-   Syntax:
-   ```
-   curl https://techsupport.enterprisedb.com/api/repository/<token>/products/tpa/release/<pgVersion>/rpm | bash
-   ```
+   - **To access the RPM repository:** Replace `token` with your company token and replace `pgVersion` with the version of Postgres you are installing.
 
-   Example:
-   ```
-   curl https://techsupport.enterprisedb.com/api/repository/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx/products/tpa/release/14/rpm | bash
-   ```
+     Syntax:
+     ```
+     curl https://techsupport.enterprisedb.com/api/repository/<token>/products/tpa/release/<pgVersion>/rpm | bash
```

Comment:
```suggestion
     curl https://techsupport.enterprisedb.com/api/repository/<token>/products/tpa/release/14/rpm | bash
```

(It doesn't actually matter at all which version you mention here, so we can remove one unneeded replacement)